cmd/oci-create-runtime-bundle/main.go

```diff
@@ -81,10 +87,20 @@ func newBundleCmd(stdout, stderr *log.Logger) *cobra.Command {
 It is strongly recommended to keep the default value.`,
 	)
 
+	cmd.Flags().BoolVar(
+		&v.version, "version", false,
+		`Print version information and exit`,
+	)
 	return cmd
 }
 
 func (v *bundleCmd) Run(cmd *cobra.Command, args []string) {
+	if v.version {
+		v.stderr.Printf("commit: %s", gitCommit)
+		v.stderr.Printf("spec: %s", specs.Version)
```

Comment:
I expect image-tools to support a range of spec versions, not just a single one.  For example, once image-spec cuts 2.0, image-tools should still be able to process 1.x images for some reasonable deprecation period.  image-spec is setting us up for that possibility by using the `v1` package for the Go types that are likely to change.  And SemVer means that if we can unpack a 1.3 image, we can also unpack 1.2, 1.1, and 1.0 images.  So I think we want this to be printing:

> specs: 0.4.0

now.  Once opencontainers/image-spec#289 lands and we bump to the new tag, we should be showing:

> specs: 0.5.0

Once 1.0 is cut we should show:

> specs: 1.0.0

Once 1.1 is cut we should show our position in the 1.x series:

> specs: 1.1.0

And then when 2.0 is cut we should show our positions in both the 1.x and 2.x series.  If 2.0 happened after 1.1 (without a 1.2, etc.), that would be:

> specs: 1.1.0, 2.0.0

And if the caller explicitly asked for the version information with `--version`, I think we should be printing it to stdout instead of stderr.